core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -39,7 +39,6 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
         FixedPoint.Unsigned rawCollateral;
     }
 
-    // TODO: determine how we should adjust collateral when the user reads it out of the contract.
```

Comment:
addressed by exposing `getCollateralPosition` which returns collateral net of fees